usersync/syncer_test.go

```diff
@@ -97,6 +98,7 @@ func TestNewSyncer(t *testing.T) {
 			givenExternalURL:    "http://syncer.com",
 			givenIFrameConfig:   iframeConfig,
 			givenRedirectConfig: redirectConfig,
+			givenForceType:      "i",
```

Comment:
I don't recommending adding this feature as part of an existing end to end test. Please add a new test at this level if you'd like to include it at this level.